### ISSUE TITLE
No welcome screen

### ISSUE THREAD

**Issue by klrock** [Aircoookie/ESP8266MinimalHTTPUpdater#1]
l have flashed the sketch to five different nodemcu's including Lolin and Amiga brands. All of them
open a WiFi Access Point "UpdateMe" which accepts the password "updrater1". However, When
"192.168.4.1" is entered into my Chrome browser, "This site can’t be reached" is displayed. Waiting as much as an hour still produces no welcome screen.

My browser "Version 78.0.3904.70 (Ofﬁcial Build) (64-bit)" and router Asus RT AC3200 work very well.
Google search has produced no applicable response to "ESP8266 Minimal HTTP Updater no welcome screen".  Is there a better search term?

Any help greatly appreciated.

**Comment by Aircoookie** [comment]
Hi, could you try a different client device (for example a phone).
I believe that your device has multiple network interfaces (e.g. LAN and WiFi) and your browser attempts to connect to 192.168.4.1 over the incorrect interface.
You can try to disable the network interface that is not WiFi. In Windows, go to Control Panel -> Network and Sharing center -> Adapter settings.

Hope that this will help :)

**Comment by klrock** [comment]
Yes, I was able to access Updater with my iPhone.  Then I activated wifi on my PC which is normally connected by ethernet.  So Now I have flashed WLED to one node mcu and am preparing to do the same on others,

Thanks so much for your excellent feedback.

**Comment by mighty-unknown** [comment]
I have a couple of the Amica NodeMCU's and I'm getting this same issue. however whenever I try to load the webpage the light flashes on the nodemcu and then the connection drops from my iPhone. 

Any thoughts on what might be happening here? 
I even powered the nodemcu with a 5v 2200mah battery pack in case the current draw was too much for the adapter I had the usb plugged into. Still no joy :(

Thanks
Ben

**Comment by xealot** [comment]
I have the issue described by @mighty-unknown exactly.

**Comment by Aircoookie** [comment]
@mighty-unknown @xealot sounds like the ESP is crashing/self resetting when you try to open the page. What Arduino core version and build settings (the ones in the "Tools" menu of Arduino IDE) are you using?

**Comment by xealot** [comment]
I'm not intimately familiar with the Arduino IDE, but these are my build settings.

![image](https://user-images.githubusercontent.com/189873/69503872-2ab81800-0eec-11ea-81ae-5a046ae9dd88.png)

Not sure where to find "Core Version".

I solved this by compiling the OTA Uploader myself using PlatformIO though. I believe I selected a newer core in the build settings there. Once that was done the pre-compiled bin file worked just fine.

**Comment by Aircoookie** [comment]
@xealot thank you for the screenshot! Seems like your core version is pretty new. I'd suspect it is 2.5.0 or 2.5.1, those versions had some occasional connectivity problems. I'd recommend 2.5.2. You could also try setting lwIP variant to v1.4 Higher bandwidth, that will also help sometimes!
Good to hear you got it working using PIO though!

**Comment by benlumley** [comment]
> @xealot thank you for the screenshot! Seems like your core version is pretty new. I'd suspect it is 2.5.0 or 2.5.1, those versions had some occasional connectivity problems. I'd recommend 2.5.2. You could also try setting lwIP variant to v1.4 Higher bandwidth, that will also help sometimes!
> Good to hear you got it working using PIO though!

I was using 2.6.0, couldn't connect. Fixed by switching to 2.5.2 and re-uploading/recompiling.
